Libraries/LibWeb/SVG/SVGElement.cpp

```diff
@@ -55,11 +55,17 @@ Optional<ARIA::Role> SVGElement::default_role() const
     // https://w3c.github.io/svg-aam/#mapping_role_table
     if (local_name() == TagNames::a && (has_attribute(SVG::AttributeNames::href) || has_attribute(AttributeNames::xlink_href)))
         return ARIA::Role::link;
-    if (local_name() == TagNames::g && should_include_in_accessibility_tree())
+    if (first_is_one_of(local_name(), TagNames::foreignObject, TagNames::g)
+        && should_include_in_accessibility_tree())
```

Comment:
Ah missed this before, but instead of `first_is_one_of`, string classes have a `is_one_of` method:
```c++
if (local_name().is_one_of(TagNames::foreignObject, TagNames::g) && should_include_in_accessibility_tree())
```

Reply:
> Ah missed this before, but instead of `first_is_one_of`, string classes have a `is_one_of` method:

Thanks much — made it do